wordpress-mobile/WordPress-Android
# Removed UC Details FF flag from wasabi build.

Since we are postponing Unified Comment details, I'm removing the FF from the wasabi build.

To test:
- Install wasabi build.
- Make sure comment details have some content and not just an empty fragment.


## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
